pkg/controller/v1beta1/inferenceservice/components/predictor/predictor.go

```diff
@@ -176,6 +180,29 @@ func (p *Predictor) CreatePredictorService(isvc *v1beta1.InferenceService) (*kns
 		},
 	}
 
+	//If InferenceService's storageUri is empty, create multi-model service configMap and
```

Comment:
What happens if the TrainedModel is created before the InferenceService? Will the trained model reconciler explode when it attempts to write to the configmap?